From 9d5bc75c87742cdd61698e0fcf144722cf2fe7fa Mon Sep 17 00:00:00 2001
From: Jaxson Han <jaxson.han@arm.com>
Date: Fri, 12 May 2023 10:07:21 +0800
Subject: [PATCH] arch: arm64: Enable FPU and FPU_SHARING for v8r aarch64

This commit is to enable FPU and FPU_SHARING for v8r aarch64.

Signed-off-by: Jaxson Han <jaxson.han@arm.com>
Upstream-Status: Backport [main: 08791d5fae2847721c871bcbc775d32a7a53b7de]
---
 arch/arm64/core/Kconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/core/Kconfig b/arch/arm64/core/Kconfig
index 968454634f78..7164e4b7b483 100644
--- a/arch/arm64/core/Kconfig
+++ b/arch/arm64/core/Kconfig
@@ -23,6 +23,9 @@ config CPU_AARCH64_CORTEX_R
 	select HAS_FLASH_LOAD_OFFSET
 	select CPU_HAS_DCACHE
 	select CPU_HAS_ICACHE
+	select CPU_HAS_FPU
+	imply FPU
+	imply FPU_SHARING
 	help
 	  This option signifies the use of a CPU of the Cortex-R 64-bit family.
 
-- 
2.34.1

